### PR TITLE
release 0.57.2

### PR DESCRIPTION
**Pull request by RenkuBot**
Release 0.57.2

This PR is auto-generated by [actions/github-script](https://github.com/actions/github-script).

/deploy

### CODE REVIEW

**Comment by leafty** [comment]
If this is going to be release, can we include https://github.com/SwissDataScienceCenter/renku-gateway/pull/726?

**Comment by olevski** [comment]
> If this is going to be released, can we include https://github.com/SwissDataScienceCenter/renku-gateway/pull/726?

@leafty yes, can you do a gateway release and add it here please?

**Comment by leafty** [comment]
@olevski Why was #3791 merged without testing? The current `HEAD` cannot be deployed:
```
Error: can't get a valid version for repositories csi-rclone. Try changing the version constraint in Chart.yaml
```
https://github.com/SwissDataScienceCenter/renku/actions/runs/11145589990/job/30975547765?pr=3782

**Comment by olevski** [comment]
> @olevski Why was #3791 merged without testing? The current HEAD cannot be deployed:

Yes we should have tested. I fixed the deployment now.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,26 @@
 .. _changelog:
 
+0.57.2
+------
+
+Renku ``0.57.2`` fixes several bugs in gateway and the `csi-rclone` driver.
+
```
Comment (rokroskar) [inline CHANGELOG.rst:7]:
```suggestion
User-facing Changes
~~~~~~~~~~~~~~~~~~~

**Bug Fixes**

- **UI**: show the correct repository access status 
- **Sessions**: allow paused sessions with cloud storage secrets to resume normally

```

Comment (rokroskar) [reply CHANGELOG.rst:7]: both of the bug fixes were for issues users saw, so I think we should mention it at the top

**Review by rokroskar — CHANGES_REQUESTED** [review]

**Review by rokroskar — APPROVED** [review]

**Review by aledegano — APPROVED** [review]

**Pull request merged**